### DISCUSSION TITLE
プレビュー画面について

### DISCUSSION THREAD

**Discussion by blono** [increments/qiita-discussions#39] [要望・アイデア・不具合]
プレビューを画面右側に一時的に収納してエディタを画面全体に出せるようにできたり、プレビュー画面でスマホやタブレットでどのような表示がされるのかを確認できると嬉しいです！

**Comment by kskwtnk** [comment]
@blono 

ご意見ありがとうございます！
パッと見て分かりづらいインターフェースなのですが……エディタを全体にするのはこちらの赤枠で囲ったボタンを押すと可能なんです🙏

| エディタとプレビュー半々 | 全画面エディタ |
| --- | --- |
| ![](https://user-images.githubusercontent.com/22670303/136494819-769b31b5-5710-43e0-af07-ca52222ba31c.png) | ![](https://user-images.githubusercontent.com/22670303/136494832-8642f309-33b8-4460-ba7e-0fcc9ed1983d.png) |

ただ、スマホやタブレットでのプレビューは存在しないですし、実機で確認している方もときどきいらっしゃるので検討しようと思います！

**Reply by blono** [reply]
ありがとうございます、確かにそのボタンでできるの確認できました！活用させていただきます！
